scapy/layers/bluetooth4LE.py

```diff
@@ -108,6 +123,11 @@ def swapbits(a):
                     state ^= lfsr_mask
         return struct.pack("<L", state)[:-1]
 
+    # def do_build(self):
+    #    #make sure post build is called
+    #    self.raw_packet_cache = None
+    #    super(Packet, self).do_build()
+
```

Comment:
What is the point of this ?

Reply:
I had some problems in the past with the CRC at the end. It is no longer needed, I will remove it.